crates/autopilot/src/infra/solvers/mod.rs

```diff
@@ -19,11 +19,29 @@ pub struct Driver {
     // winning solution should be discarded if it contains at least one order, which
     // another driver solved with surplus exceeding this driver's surplus by `threshold`
     pub fairness_threshold: Option<eth::Ether>,
+    pub submission_address: eth::Address,
     client: Client,
 }
 
 impl Driver {
-    pub fn new(url: Url, name: String, fairness_threshold: Option<eth::Ether>) -> Self {
+    pub async fn new(
+        url: Url,
+        name: String,
+        fairness_threshold: Option<eth::Ether>,
+        submission_account: Account,
+    ) -> Self {
+        let submission_address = match submission_account {
+            Account::Kms(key_id) => {
+                let config = ethcontract::aws_config::load_from_env().await;
+                let account =
+                    ethcontract::transaction::kms::Account::new((&config).into(), &key_id.0)
+                        .await
+                        .unwrap_or_else(|_| panic!("Unable to load KMS account {:?}", key_id));
```

Comment:
I would feel more safe to `trace::error` here instead of panicking. At least some drivers will run until we figure out why the misconfiguration, instead of having a crashloop.